From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Mon, 17 Oct 2016 20:24:33 +0200
Subject: [PATCH] {MAYBE UPS} nfs: make clang happy

clang reports:

    fs/nfs/nfs4proc.c:1548:25: error: suggest braces around
    initialization of subobject [-Werror,-Wmissing-braces]
            nfs4_stateid freeme = {0};
                                   ^
                                   {}
---
 fs/nfs/nfs4proc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/nfs/nfs4proc.c b/fs/nfs/nfs4proc.c
index ad917bd72b38..ad31c51aa9b3 100644
--- a/fs/nfs/nfs4proc.c
+++ b/fs/nfs/nfs4proc.c
@@ -1545,7 +1545,7 @@ static int update_open_stateid(struct nfs4_state *state,
 	struct nfs_client *clp = server->nfs_client;
 	struct nfs_inode *nfsi = NFS_I(state->inode);
 	struct nfs_delegation *deleg_cur;
-	nfs4_stateid freeme = {0};
+	nfs4_stateid freeme = {};
 	int ret = 0;
 
 	fmode &= (FMODE_READ|FMODE_WRITE);
-- 
